[0m[[0m[31merror[0m] [0m[0m/home/nasmth/projects/project0/project0/src/main/scala/project0/DAO.scala:60:11: not found: value conn[0m
[0m[[0m[31merror[0m] [0m[0m          conn.close()[0m
[0m[[0m[31merror[0m] [0m[0m          ^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
